rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] A rental belongs to a book and also belongs to a person: 🤔 So your initialize method should look something like below ⬇️
```ruby
def initialize(date, book, person)
    @book = book
    book.rentals << self
    @person = person
    person.rentals << self
  end
```